### ISSUE TITLE
Symlink "locked" icon

### ISSUE THREAD

**Issue by cassidyjames** [pop-os/icon-theme#28]
We use a `locked` icon in the installer overlaid on a drive for drive encryption. There may be a better namespace—though nothing was obvious from the icon naming spec. We could use something like `dialog-password` if @danrabbit is okay with the semantic change, but I think a "locked" metaphor is more accurate than a "password" metaphor.

Right now in Pop! the `locked` icon falls back to a GNOME-style icon. In Pop! we could just symlink to one of the existing lock icons.

## Current:
![image](https://user-images.githubusercontent.com/611168/35457686-0337e674-0297-11e8-8d94-2377d7134324.png)

## More Ideal:
![image](https://user-images.githubusercontent.com/611168/35457666-f3de3dfe-0296-11e8-9e62-ec8bac2dbf63.png)

**Comment by danrabbit** [comment]
Another option might be "channel-secure"

**Comment by cassidyjames** [comment]
Welp, we changed to using the FD.o `security-high` in the installer which also works well in Pop!

![image](https://user-images.githubusercontent.com/611168/35458550-0867a938-029a-11e8-850b-eefb5bec17d9.png)

It might still be worth symlinking, but it's less of a big deal now.

**Comment by WatchMkr** [comment]
I imagine this will move to an image that Kate creates matching the rest of the Pop installer artwork.

**Comment by cassidyjames** [comment]
Yeah, I had mentioned that in Slack, but using more standardized icon names in both the installer and our icon theme is a good goal.
